src/_scss/pages/recipient/recipientPage.scss

```diff
@@ -1,7 +1,10 @@
 .usa-da-recipient-page {
 	@import "all";
 	@import "layouts/default/default";
-    @import "layouts/default/stickyHeader/header";
+  @import "layouts/default/stickyHeader/header";
+
+	@import "./topFive/topFive";
```

Comment:
Since the styling is mostly the same, you can import `"pages/state/topFive/topFive"` and override/ add styling as needed.